aiosmtpd/smtp.py

```diff
@@ -243,15 +243,10 @@ def connection_lost(self, error):
     def eof_received(self):
         log.info('%r EOF received', self.session.peer)
         self._handler_coroutine.cancel()
-        if self.session.ssl is not None:            # pragma: nomswin
```

Comment:
No longer needed thanks to `test_starttls:TestTLSEnding:test_eof_received`, which works on Windows.